src/main/java/com/flab/offcoupon/dto/response/MonthlyOrderStatistics.java

```diff
@@ -0,0 +1,28 @@
+package com.flab.offcoupon.dto.response;
+
+import com.flab.offcoupon.domain.vo.persistence.statistics.MonthlyOrderStatisticsVo;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+import java.time.YearMonth;
+
+
+@Getter
+@AllArgsConstructor
+public final class MonthlyOrderStatistics {
+
+    private final YearMonth yearMonth;
+    private final long totalOrderCnt;
+    private final BigDecimal totalPaymentPrice;
+    private final long totalCouponUseCnt;
+    private final BigDecimal totalCouponPrice;
+
+    public MonthlyOrderStatistics(MonthlyOrderStatisticsVo vo) {
+        this.yearMonth = (vo != null) ? vo.yearMonth() : null;
+        this.totalOrderCnt = (vo != null) ? vo.totalOrderCnt() : 0;
+        this.totalPaymentPrice = (vo != null) ? vo.totalPaymentPrice() : BigDecimal.ZERO;
+        this.totalCouponUseCnt = (vo != null) ? vo.totalCouponUseCnt() : 0;
+        this.totalCouponPrice = (vo != null) ? vo.totalCouponPrice() : BigDecimal.ZERO;
+    }
```

Comment:
생성자 파라미터로 null이 넘어와야하는 경우가 있나요? 만약 null이 넘어온다면 오류인 상황일 것으로 보이는데, 이 경우에는 명시적으로 에러를 throw해주는게 좋습니다. 이런식으로 단언문을 사용하기도 하니 참고해주세요~
```suggestion
    public MonthlyOrderStatistics(MonthlyOrderStatisticsVo vo) {
      Assert.notNull(vo, "생성자 파라미터는 null일 수 없습니다");
      ...
    }
```

Reply:
이 부분은 제가 조회에서 데이터 가져올때 어차피 1개월 기준으로 받아오니까 List가 아닌 객체로 받으면 되지 않을까 싶었는데요!
List가 아닌 객체로 받아오니까 `해당하는 기간 범위에 데이터가 없을때`에 객체가 Null이 되버리는 문제가 있어서 추가했었습니다!

그런데, List로 받아오면 애초에 null인 객체가 List에 추가되지 않아서 List로 반환하는걸로 했습니다.
그래서 말씀해주신 Null 체크는 없애도록 하겠습니다! 더 좋은 코드 추천해주셔서 감사합니다~!